doc/Getting_Started.md

```diff
@@ -61,6 +63,7 @@ $ pip3 install --upgrade tockloader
 
 Ubuntu:
 ```
+$ apt install -y build-essential gcc-arm-none-eabi python3-pip curl
```

Comment:
While this is definitely useful for fast getting started, it is only necessary of you are building for ARM.

```suggestion
# If planning to build for ARM
$ apt install -y build-essential gcc-arm-none-eabi python3-pip curl
```

Reply:
How about this instead;

```suggestion
$ apt install -y build-essential python3-pip curl
# If planning to build for ARM
$ apt install -y gcc-arm-none-eabi
```

I don't know exactly, but I think it's unlikely that the host portions (build-essential) are dependent on the target architecture.

Reply:
BTW, reactions don't show up in email, so if I didn't check the thread using web UI, I would have thought this comment was forgotten.